<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningDisDelSureDialog</class>
 <widget class="QDialog" name="WarningDisDelSureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid rgb(200,200,200);
border-radius:5px;
background-color:rgb(255,255,255)</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
background-color:rgb(255,0,0)</string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>删除确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>3</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="contentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
